apps/ssr-tests-v9/src/build.ts

```diff
@@ -28,6 +28,7 @@ async function build() {
   const generateStartTime = process.hrtime();
 
   const rawStoriesGlobs = getVnextStories() as string[];
+  rawStoriesGlobs.push(path.resolve(path.join(__dirname, './stories/**/index.stories.tsx')));
```

Comment:
Should it simply use a function from `apps/public-docsite-v9/.storybook/main.utils.js`? As I see a glob there includes `index.stories.tsx`

Reply:
that utility is still being used, but does not cover `apps/` and in this case we **do not want it to cover apps** because these stories should only be used for SSR tests and not a part of our docsite

Reply:
Yeah, overlooked initially from where these stories are coming 🙃